examples/test_vault.ipynb

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
<p>It's definitely a matter of opinion so you don't have to take this one</p><p>IMO the code will be much more readable if you'll simply <code>import mlrun</code>  and then just do <code>mlrun.new_task</code>,  <code>mlrun.new_project</code>  etc...</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mlrun/mlrun/pull/500/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>I think I'll do that, I see that I import a lot of stuff I no longer use (run_local, get_run_db etc.) - it's better if I just <code>import mlrun</code>   and use what I need to use specifically.</p>